applications/teletype/src/teletype_util.erl

```diff
@@ -785,6 +785,7 @@ find_account_id(JObj) ->
 find_account_db(<<"account">>, JObj) -> find_account_db_from_id(JObj);
 find_account_db(<<"user">>, JObj) -> find_account_db_from_id(JObj);
 find_account_db(<<"fax">>, JObj) -> find_account_db(JObj);
+find_account_db(<<"port_request">>, _JObj) -> <<"port_requests">>;
```

Comment:
@jamesaimonetti what do you think to default to  find_account_db(JObj) ?
@macpie if the port_request sets the Account-DB in the notification you can can call find_account_db